--- new-poc-instance/temperature/Dockerfile	2019-08-21 16:44:15.000000000 +0100
+++ ../vagrant-poc/PoCx86/temperature/Dockerfile	2019-07-25 11:05:01.000000000 +0100
@@ -1,16 +1,6 @@
-FROM arm32v6/alpine:latest
-
-RUN apk update &&\
-    apk upgrade && \
-    apk add --no-cache --virtual .build-deps alpine-sdk glib-dev git && \
-    apk add --no-cache bash python2-dev py-pip supervisor && \
-    pip install --upgrade pip && \
-    pip install --no-cache-dir paho-mqtt git+https://git.research.arm.com/rsh/emerge/health-poc/debug-tools.git#egg=debug_tools && \
-    apk del .build-deps
+FROM pihealth_trimmed
 
 COPY read_temp.py /read_temp.py
 COPY supervisord.conf /etc/supervisord.conf
 
-RUN  mkdir -p /var/log/supervisord
-
-CMD [ "supervisord", "-n", "-c", "/etc/supervisord.conf" ]
\ No newline at end of file
+CMD [ "supervisord", "-n", "-c", "/etc/supervisord.conf" ]
