--- new-poc-instance/context/Dockerfile	2019-08-21 16:43:52.000000000 +0100
+++ ../vagrant-poc/PoCx86/context/Dockerfile	2019-08-21 14:13:35.000000000 +0100
@@ -1,10 +1,10 @@
-FROM resin/rpi-raspbian:stretch
+FROM debian
 
 COPY precise-engine_0.2.0_armv7l.tar.gz /tmp/precise-engine_0.2.0_armv7l.tar.gz
 
 RUN apt-get update &&\
-    apt-get upgrade && \
-    apt-get install -y --no-install-recommends wget gettext portaudio19-dev supervisor python3-dev python3-pip pulseaudio pulseaudio-utils python3-blinkt supervisor python3-setuptools build-essential && \
+    apt-get -y upgrade && \
+    apt-get install -y --no-install-recommends wget gettext portaudio19-dev supervisor python3-dev python3-pip pulseaudio pulseaudio-utils supervisor python3-setuptools build-essential && \
     pip3 install --no-cache-dir paho-mqtt requests precise-runner && \
     tar xvf /tmp/precise-engine_0.2.0_armv7l.tar.gz && \
     mkdir -p var/log/supervisord && \
