--- new-poc-instance/vitals/Dockerfile	2019-08-21 16:44:17.000000000 +0100
+++ ../vagrant-poc/PoCx86/vitals/Dockerfile	2019-07-25 11:05:01.000000000 +0100
@@ -1,17 +1,12 @@
-FROM arm32v6/alpine:3.8
+FROM pihealth_trimmed
 
 ENV BLE_TARGET_BP "c8:fd:19:3a:a2:5f"
 
 #install prerequisites
-RUN apk add --no-cache --virtual .build-deps alpine-sdk && \
-    apk add --no-cache bash glib-dev linux-headers python-dev py-pip supervisor bluez-deprecated && \
-    pip install --no-cache-dir bluepy paho-mqtt requests && \
-    mkdir -p /var/log/supervisord && \
-    apk del .build-deps
+RUN apk del .build-deps
 
 COPY supervisord.conf /etc/supervisord.conf
 COPY blue_bp /blue_bp
 
 #do startscript
 CMD [ "supervisord", "-n", "-c", "/etc/supervisord.conf" ]
-
